base_layer/core/src/base_node/state_machine_service/initializer.rs

```diff
@@ -81,7 +81,7 @@ where B: BlockchainBackend + 'static
 
     fn initialize(&mut self, context: ServiceInitializerContext) -> Self::Future {
         trace!(target: LOG_TARGET, "init of base_node");
-        let (state_event_publisher, _) = broadcast::channel(10);
+        let (state_event_publisher, _) = broadcast::channel(500);
```

Comment:
This is not a huge change so I think its fine.

Reply:
Yes, still seing a lot of `[c::bn::state_machine_service::states::listening] DEBUG Metadata event subscriber lagged` in my testing.

Reply:
Sorry I disagree. If the base node has changed state 10 times and there is a part of the system that has not read the event, it is probably a sign that something is broken. I think leave it as 10

Reply:
My argument is that these events should not be missed, as some parts of the system depends on all the state events. They could be missed for various reasons, one being if the base node has to defend against many spam messages, as happended just recently with join messages. It is much better to be resilient in this regard, as opposed to some part of the system not working properly. The default value of 10 was just a guess at the time, and now we have seen that it can sometimes be inadequate. As the tokio broadcast channels are also self-regulating i.t.o. memory usage, its use will be optimal.

Reply:
The `ridcully` testnet also still has `[c::bn::state_machine_service::states::listening] DEBUG Metadata event subscriber lagged` log entries.